Sources/SwiftInspectorVisitors/ExtensionVisitor.swift

```diff
@@ -60,19 +60,13 @@ public final class ExtensionVisitor: SyntaxVisitor {
       declarationModifierVisitor.walk(modifiers)
     }
 
-    let propertyVisitor = PropertyVisitor()
-    propertyVisitor.walk(node.members)
```

Comment:
another benefit of not walking the entire declaration node here is that our property visitor doesn't need to be hardened against accidentally reading properties from nested classes (doesn't hurt to leave that hardening but this approach still leaves us with fewer ways we can hurt ourselves)